### PR TITLE
Added system proxy configuration option

### PR DESCRIPTION
**Pull request by mikedgibson**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Per customer request we have added the ability to use the system proxy for communication with the Trend Micro Vision One APIs from our integration

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [X] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by AradCarmi** [comment]
Hi @mikedgibson ,
Thank you for contributing!
Apologies for the delay in reviewing your PR, the team will review your code as soon as possible.

Thanks again.

File: Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOne/TrendMicroVisionOne.py [hunk 1]
```diff
@@ -167,23 +167,25 @@ def check_datetime_aware(d):
...
-        :param verify: Whether the request should verify the SSL certificate.
+        :type proxy: ``bool``
+        :param proxy: Whether the request should use the system proxy settings.
         :return: returns None
         :rtype: ``None``
         """
-        super().__init__(base_url=base_url)
         self.base_url = base_url
         self.api_key = api_key
+        self.proxy = proxy
```
Comment (MosheEichler) [inline Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOne/TrendMicroVisionOne.py:184]: Why do you need this?
```suggestion
```

**Review by MosheEichler — COMMENTED** [review]
Hi @Mike.
You did an Excellent Job!! :)

Please check the following notes.
And let me know when you are done.

Thank you.

**Review by MosheEichler — APPROVED** [review]

File: Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOne/TrendMicroVisionOne.py (hunk 1 above)
Comment (MosheEichler) [reply Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOne/TrendMicroVisionOne.py:184]: Nope. its ok I'll fix it.
I just wanted to make sure it was really unnecessary :).

Thanks!

**Pull request merged**